### ISSUE TITLE
Infer question

### ISSUE THREAD

**Issue by kimtaehyeong** [RuiyangJu/Fracture_Detection_Improved_YOLOv8#1]
Hello.
Thank you for researching a good project.
I have a question. The training code is provided, but the inference code does not seem to be provided. Is it possible to get the box coordinates by inference like normal yolo?

**Comment by RuiyangJu** [comment]
@kimtaehyeong Hello! Thank you for your interest in our project. The training code we provide includes inference. In fact, after training, our code will automatically select the best.pt among all Epochs for inference. The example is as follows:

> Validating runs\detect\train\weights\best.pt...
> Ultralytics YOLOv8.0.147  Python-3.8.18 torch-2.1.1 CUDA:0 (NVIDIA GeForce RTX 3090, 24576MiB)
> Model summary (fused): 273 layers, 44663659 parameters, 0 gradients, 166.3 GFLOPs
>                  Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 171/171 [01:25<00:00,  2.01it/s]
>                    all       4094       9613      0.621      0.653      0.628      0.409
>            boneanomaly       4094         53      0.188      0.096      0.105     0.0371
>             bonelesion       4094          8      0.434        0.5      0.357      0.212
>               fracture       4094       3740      0.892       0.92      0.954      0.586
>                  metal       4094        168      0.907      0.905      0.942      0.815
>     periostealreaction       4094        697      0.649      0.709      0.716      0.372
>           pronatorsign       4094        104      0.528      0.798      0.693      0.369
>             softtissue       4094         89      0.412      0.315      0.265      0.133
>                   text       4094       4754      0.962      0.984      0.991      0.751
> Speed: 0.2ms preprocess, 14.7ms inference, 0.0ms loss, 0.7ms postprocess per image
> Saving runs\detect\train\predictions.json...
> Results saved to runs\detect\train

**Comment by kimtaehyeong** [comment]
Thank you so much. I solved problem!
